src/data/html-quiz.ts

```diff
@@ -1,4 +1,16 @@
 const htmlQuiz = [
+  {
+    Question:
+    "Is HTML an Programming Languaje? Why?",
+    Answer: "No, becase HTML don't have dinamic functionalities",
+    Distractor1: "Yes, we can create interactive content to users",
+    Distractor2: "No, because we can't compilate",
+    Distractor3: "Yes, we can insert forms, media and another functionalities",
+    Explanation:
+      "In HTML we can structuring and create sections, paragraphs and links, below elements, labels and attributes",
+    Link:
+      "https://www.hostinger.com/tutorials/what-is-html"  
```

Comment:
Let's remove this question.
We want to avoid questions that only yield yes or no answers 👍

Reply:
ok, remove the question

Reply:
> ok, remove the question

Since this is your PR, you will need to remove this before another review can take place

Reply:
Done